cpp_linter/rest_api/github_api.py

```diff
@@ -37,6 +37,71 @@
     retry="retry-after",
 )
 
+QUERY_REVIEW_COMMENTS = """
+ query($owner: String!, $name: String!, $number: Int!) {
+     repository(owner: $owner, name: $name) {
+         pullRequest(number: $number) {
+            id
+            reviewThreads(last: 100) {
+                nodes {
+                    id
+                    isResolved
+                    isCollapsed
+                    comments(first: 100) {
+                        nodes {
+                            id
+                            body
+                            path
+                            line
+                            startLine
+                            originalLine
+                            originalStartLine
+                            author {
+                                login
+                            }
```

Comment:
Can probably remove this author field. The info is rather useless to us.

Reply:
yep, its now no longer used so its been removed